README.md

```diff
@@ -18,11 +18,12 @@
 
 ## Overview
 
-This package is a python implementation of the [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) method [1]
-for differential expression analysis (DEA) with bulk RNA-seq data, originally in R.
+This package is a python implementation of the [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) 
+method [1] for differential expression analysis (DEA) with bulk RNA-seq data, originally in R.
```

Comment:
not a scope of this PR but would be nice if there was a link to [1] leading to the references in the bottom of the page